src/Server/WebServer.fs

```diff
@@ -1,149 +1,14 @@
-            let positive = getAs<string> adjective.Positive
-            let comparatives = getAs<string []> adjective.Comparatives
-            Task(positive, comparatives)
-    
-        let task = adjective |> Option.map getTask |> Option.toObj 
-        return! Successful.OK task next ctx
-    }
-
-let getVerbImperativesTask next (ctx : HttpContext) =
-    task {
-        let classFromQuery = ctx.GetQueryStringValue "class"
-        let classFilter = getAzureFilter "Class" Int classFromQuery
-
-        let patternFromQuery = ctx.GetQueryStringValue "pattern"
-        let patternFilter = getAzureFilter "Pattern" String patternFromQuery
-
-        let filters = 
-            [ classFilter; patternFilter ]
-            |> Seq.choose id
-            
-        let verb = tryGetRandom<VerbImperative.VerbImperative> "verbimperatives" filters
-
-        let getTask (verb: VerbImperative.VerbImperative) = 
-            let indicative = getAs<string> verb.Indicative
-            let imperatives = getAs<string []> verb.Imperatives
-            Task(indicative, imperatives)
-
-        let task = verb |> Option.map getTask |> Option.toObj 
-        return! Successful.OK task next ctx
-    }
-
-let getVerbParticiplesTask next (ctx: HttpContext) =
-    task {
-        let patternFromQuery = ctx.GetQueryStringValue "pattern"
-        let patternFilter = getAzureFilter "Pattern" String patternFromQuery
-
-        let regularityFromQuery = ctx.GetQueryStringValue "isRegular"
-        let regularityFilter = getAzureFilter "IsRegular" Bool regularityFromQuery
-
-        let filters =
-            [ patternFilter; regularityFilter ] 
-            |> Seq.choose id
-        
-        let verb = tryGetRandom<VerbParticiple.VerbParticiple> "verbparticiples" filters
-
-        let getTask (verb: VerbParticiple.VerbParticiple) = 
-            let infinitive = getAs<string> verb.Infinitive
-            let participles = getAs<string []> verb.Participles
-            Task(infinitive, participles)
-
-        let task = verb |> Option.map getTask |> Option.toObj 
-        return! Successful.OK task next ctx
-    }
+open Tasks
 
 let webApp = router {
-    get "/api/nouns/plurals" getNounPluralsTask
-    get "/api/nouns/accusatives" getNounAccusativesTask
-    get "/api/adjectives/plurals" getAdjectivePluralsTask
-    get "/api/adjectives/comparatives" getAdjectiveComparativesTask
-    get "/api/verbs/imperatives" getVerbImperativesTask
-    get "/api/verbs/participles" getVerbParticiplesTask
+    get "/api/nouns/plurals" Noun.getNounPluralsTask
+    get "/api/nouns/accusatives" Noun.getNounAccusativesTask
+    get "/api/adjectives/plurals" Adjective.getAdjectivePluralsTask
+    get "/api/adjectives/comparatives" Adjective.getAdjectiveComparativesTask
+    get "/api/verbs/imperatives" Verb.getVerbImperativesTask
+    get "/api/verbs/participles" Verb.getVerbParticiplesTask
+    get "/api/verbs/conjugations" Verb.getVerbConjugationTask
```

Comment:
Other exercises are called after grammatical forms so plural is applicable whereas conjugation is a name for the grammar category which sounds strange in plural... I would either suggest this
```suggestion
    get "/api/verbs/conjugation" Verb.getVerbConjugationTask
```
or just having singular for all the endpoints.